### ISSUE TITLE
OsmAnd does not show any map to download

### ISSUE THREAD

**Issue by tps800** [osmandapp/OsmAnd#19661]
### Description

OsmAnd 4.6.14 does not show any map to download in "Karten & Resourcen -> Downloads".

### Steps to reproduce

Upgrade or install OsmAnd from available repositories like Google Store, F-Droid, then try to download maps. OsmAnd wont show any after loading repositories.

### Actual result

No maps found

### Expected result

Maps found

### Your Environment (required)

```
OsmAnd Version: 4.6.14
Android/iOS version: 6.0.1
Device model: SM-N910F
Crash-Logs: none
```
/

**Comment by dmpr0** [comment]
Fixed in 4.7.10. Please also notice that 4.7.10 is the last version that supports Android 6.

**Comment by ohhai** [comment]
Just realized that OSMAnd+ stopped updating at my old phone.
Currently have 4.6.13 and the subject problem as well.

Is there any way to get official prebuilt 4.7.10 for Android 6?
Seems like Google Play does not offer old versions.
